src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.Month;
+
+public class Deadline extends Task {
```

Comment:
You've avoided a common mistake by making sure there is a white spacing between the class name "Task" and '{' as many overlooked this. 👍